### DISCUSSION TITLE
Account switching for particular RP

### DISCUSSION THREAD

**Discussion by dselyuzhitskiy** [ory/hydra#3063] [Q&A]
Hi!

I'm trying to find a way to let user change account for particular RP without deleting or changing current openid session. As i understand, prompt=select_account is not supported, so is there are any another way to do it?

For example, i could use prompt=login and force render login page but i want to detect using prompt in our login app to prevent remembering new session ( which cause setting oauth2_authentication_session ) because, as i know, new session will be used in other RP.

**Comment by vinckr** [comment]
Hello @dselyuzhitskiy 
apologies for the late answer. 
Can you elaborate a bit more on the use case? The user is logged in with say Facebook OIDC and wants to switch to e.g the GitHub OIDC ? 
This might be more a use case for [Ory Kratos](https://github.com/ory/kratos/), but maybe I misunderstand what you are trying to achieve.
